dashboard/lib/services/README.md

```diff
@@ -0,0 +1,25 @@
+# Service Objects
+
+This directory is meant to contain Plain Old Ruby Objects (POROs) that
+encapsulate business logic which _does something_. This is as
+opposed to other categories of POROs like Policy Objects which _tell you about something_
+or Query Objects which _retrieve something_.
+
+Service Objects declared in this directory should all be declared under the
+`Services` namespace and should contain non-Rails-specific business logic that
+can be used by our Rails app. In particular, this is a great place to put code
+that would otherwise end up on our Rails models or controllers but which
+doesn't necessarily _need_ to be declared there.
+
+## More Info
+
+See the [original pull request][original PR] for more info and discussion about
+this pattern. Of particular interest in that pull request are [this
+article][refactor article] detailing a philosophy of categorizing different
```

Comment:
Just curious: what is the double `[x][y]` syntax?

Reply:
They're called [reference-style links](https://daringfireball.net/projects/markdown/syntax#link); it's a syntax that lets you refer to a link URL elsewhere in the document instead of placing it inline in your text.  It's fairly common to list them like endnotes.  This is the format Elijah chose for the existing READMEs I based this one on, so I kept it.